django/db/models/fields/__init__.py

```diff
@@ -92,6 +92,10 @@ def _empty(of_cls):
     return new
 
 
+def return_None():
```

Comment:
Not faster because it's locally cached but reduces memory usage by creating only one function, `lambda: None` in each invocation creates a new one for each field (though they share the same code object)